src/App.js

```diff
@@ -1,43 +1,25 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, {useState} from "react";
 import "./App.css";
-import BottomRow from "./BottomRow";
+import Scoreboard from "./Scoreboard.js";
+import Buttons from "./Buttons.js";
+// import Scoreboard, {homeScore, setHomeScore, awayScore, setAwayScore} from "./Scoreboard.js";
+
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
 
+  //setting state variables
+  const [homeScore, setHomeScore] = useState(0) 
+  const [awayScore, setAwayScore] = useState(0)
+
   return (
     <div className="container">
-      <section className="scoreboard">
-        <div className="topRow">
-          <div className="home">
-            <h2 className="home__name">Lions</h2>
-
-            {/* TODO STEP 3 - We need to change the hardcoded values in these divs to accept dynamic values from our state. */}
-
-            <div className="home__score">32</div>
-          </div>
-          <div className="timer">00:03</div>
-          <div className="away">
-            <h2 className="away__name">Tigers</h2>
-            <div className="away__score">32</div>
-          </div>
-        </div>
-        <BottomRow />
-      </section>
-      <section className="buttons">
-        <div className="homeButtons">
-          {/* TODO STEP 4 - Now we need to attach our state setter functions to click listeners. */}
-          <button className="homeButtons__touchdown">Home Touchdown</button>
-          <button className="homeButtons__fieldGoal">Home Field Goal</button>
-        </div>
-        <div className="awayButtons">
-          <button className="awayButtons__touchdown">Away Touchdown</button>
-          <button className="awayButtons__fieldGoal">Away Field Goal</button>
-        </div>
-      </section>
+      <Scoreboard homeScore={homeScore} setHomeScore={setHomeScore} awayScore={awayScore} setAwayScore = {setAwayScore}/>
+      <Buttons homeScore={homeScore} setHomeScore={setHomeScore} awayScore={awayScore} setAwayScore = {setAwayScore} />
```

Comment:
Instead of giving these components direct access to `setAwayScore`, we could write the function we will actually use in there, like the one for touchdowns, and pass that as a prop. That way, we keep our components 'dumb'. They don't need to contain logic outside of their direct responsibilities.